### DISCUSSION TITLE
Something is adding odd subdomains in GitHub Pages

### DISCUSSION THREAD

**Discussion by FANMixco** [community/community#50507] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Recently, Google reported a Soft 404 on my website:
https://federiconavarrete.com

The odd subdomain is:
http://gaicheisi.federiconavarrete.com/

That displays some Russian texts:
<img width="616" alt="image" src="https://user-images.githubusercontent.com/1666156/226386529-e5179c90-36cd-4584-a6f5-0d5c99eda363.png">

I haven't created a subdomain on my provider as the website doesn't have any and this is the provider's answer:
<img width="247" alt="image" src="https://user-images.githubusercontent.com/1666156/226386987-09fbf6d5-077e-4ff2-830e-67e411be46e4.png">

I'd say something odd is happening in GitHub pages.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @FANMixco ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by yoannchaudet** [comment]
It looks to me you have pointed a wildcard `*` DNS record at GitHub Pages on your DNS zone. That means anyone on the platform is currently capable of taking over any of your subdomains 😬.

I strongly suggest you remove the wildcard DNS record asap. We call it out in our documentation: https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-a-subdomain (red warning).
